src/main/kotlin/no/nav/syfo/application/util/JaxBUtils.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.application.util
+
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.dataformat.xml.XmlMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import com.migesok.jaxb.adapter.javatime.LocalDateTimeXmlAdapter
+import com.migesok.jaxb.adapter.javatime.LocalDateXmlAdapter
+import javax.xml.bind.JAXBContext
+import javax.xml.bind.Marshaller
+import javax.xml.bind.Marshaller.JAXB_ENCODING
+import javax.xml.bind.Unmarshaller
+import no.nav.helse.eiFellesformat.XMLEIFellesformat
+import no.nav.helse.msgHead.XMLMsgHead
+import no.nav.helse.sm2013.HelseOpplysningerArbeidsuforhet
+
+val xmlObjectWriter: XmlMapper = XmlMapper().apply {
+    registerModule(JavaTimeModule())
+    registerKotlinModule()
+    configure(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS, false)
+}
+
+val fellesformatJaxBContext: JAXBContext = JAXBContext.newInstance(
+    XMLEIFellesformat::class.java,
+    XMLMsgHead::class.java,
+    HelseOpplysningerArbeidsuforhet::class.java
+)
+val fellesformatUnmarshaller: Unmarshaller = fellesformatJaxBContext.createUnmarshaller().apply {
```

Comment:
Unmarshaller må created for hver ny request, JAXBContext er threadsafe, men ikke Unmarshaller og Marshaller